pkg/daemon/ceph/client/osd.go

```diff
@@ -180,6 +180,26 @@ func EnableScrubbing(context *clusterd.Context, clusterName string) (string, err
 	return string(buf), nil
 }
 
+//SetNoOut execs `ceph osd set noout`
+func SetNoOut(context *clusterd.Context, clusterName string) (string, error) {
+	args := []string{"osd", "set", "noout"}
+	buf, err := ExecuteCephCommand(context, clusterName, args)
+	if err != nil {
+		return string(buf), fmt.Errorf("failed to set noout: %+v", err)
+	}
+	return string(buf), nil
+}
+
+//UnsetNoOut execs `ceph osd unset noout`
```

Comment:
```suggestion
// UnsetNoOut execs `ceph osd unset noout`
```

Reply:
Updating all comments to start with a space.